index.d.ts

```diff
@@ -0,0 +1,11 @@
+import { Model, ModelClass } from 'objection';
+
+declare module 'objection-graphql-relay' {
+  export function idWrapper(modelName?: string): (parent: Model) => Promise<string>;
+  export function fromGlobalId(modelName?: string, globalId?: string): string;
+  export function singleRelationshipWrapper(relationshipName: string): Promise<string>;
+  export function connectionWrapper(field: string | object): Promise<object>;
```

Comment:
Probably better instead of just typing to `object` to create the interface representing the possible configuration values. Something like this:

```typescript
interface ConnectionWrapperResults {
  results: any[]
  total: number
}
type ConnectionWrapperField = string | any[] | ConnectionWrapperResults
interface Edge {
  cursor: string
  node: Model
}
interface Connection {
  pageInfo {
    hasPreviousPage: boolean
    hasNextPage: boolean
  }
  edges: Edge[]
  totalCount: number
}

export function connectionWrapper(field: ConnectionWrapperField): Connection
```